go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/pkg/errors v0.9.1
-	github.com/relationalai/rai-sdk-go v0.4.7-alpha
+	github.com/relationalai/rai-sdk-go v0.5.0-alpha
```

Comment:
need to fix inconsistent vendoring issue (see [GHA workflow failure](https://github.com/RelationalAI/rai-cli/actions/runs/4930887139/jobs/8946070152?pr=58))

Reply:
updated